src/components/Login/SignIn.jsx

```diff
@@ -308,7 +311,7 @@ export default function SignIn({ isLoggedIn, setIsLoggedIn, wrapperSetLoading =
               Sign In
             </Typography>
             <Typography align="left" className={classes.subtext} variant="body1" gutterBottom>
-              Welcome back! Please enter your details.
+              Welcome back! Please login.
```

Comment:
I'm in favor of changing the wording on this as suggested here, as "enter details" does not make sense in use cases where zot does not have username/password login enabled. But need @andaaron or @rchincha confirmation on if they are ok with the change.

Reply:
That was exactly my thought ;)

Reply:
The wording is fine. However, there is a ci/cd failure!

Reply:
It seems that timeout on integration tests are fairly common if I take a look at other merged PR.
Not sure it's related to this code change :/

Reply:
Generally we aim for the integration tests to pass before tagging a new zui release. I'll try to patch them up today so you can run them against this PR.